mllib/src/test/java/org/apache/spark/mllib/regression/JavaIsotonicRegressionSuite.java

```diff
@@ -81,10 +81,10 @@ public void testIsotonicRegressionPredictionsJavaRDD() {
     JavaDoubleRDD testRDD = sc.parallelizeDoubles(Arrays.asList(0.0, 1.0, 9.5, 12.0, 13.0));
     List<Double> predictions = model.predict(testRDD).collect();
 
-    Assert.assertTrue(predictions.get(0) == 1d);
-    Assert.assertTrue(predictions.get(1) == 1d);
-    Assert.assertTrue(predictions.get(2) == 10d);
-    Assert.assertTrue(predictions.get(3) == 12d);
-    Assert.assertTrue(predictions.get(4) == 12d);
+    Assert.assertEquals(1.0, predictions.get(0).doubleValue(), 1.0e-14);
```

Comment:
No it's still asserting equality to 1.0; the expected value comes first so that's where 1.0 went (changed from the more obscure 1d). I added the epsilon at the end since the version without it is deprecated. (And I just made up a tiny epsilon.)